addons/point_of_sale/static/src/js/models.js

```diff
@@ -1401,7 +1401,6 @@ exports.Orderline = Backbone.Model.extend({
         this.set_quantity(1);
         this.discount = 0;
         this.discountStr = '0';
-        this.type = 'unit';
```

Comment:
you can probably also remove 'product_type' from ProductCategoriesWidget that was probably lminked to this feature

Reply:
Looks like it is a filter to show only weighted products, but indeed never used so ... removed!